### PR TITLE
chore:add visibility level to artwork

### PR DESCRIPTION
**Pull request by nspinazz89**
This PR adds visibility level to artwork per [PX-5249]. It also has a related [gravity ticket ](https://github.com/artsy/gravity/pull/15693)that needs to be merged first.

[PX-5249]: https://artsyproduct.atlassian.net/browse/PX-5249?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/schema/v2/artwork/index.ts [hunk 1]
```diff
@@ -1446,6 +1446,11 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           }
         },
       },
+      visibilityLevel: {
+        description: "The visibility level of the artwork",
+        type: GraphQLString,
+        resolve: ({ visibility_level }) => visibility_level,
+      },
```
Comment (jpotts244) [inline src/schema/v2/artwork/index.ts:1453]: Might be nice to add a spec to capture the new field

Something simple like:
https://github.com/artsy/metaphysics/blob/main/src/schema/v2/artwork/__tests__/artwork.test.js#L692-L714 for example!

File: _schemaV2.graphql [hunk 2]
```diff
@@ -2171,6 +2171,9 @@ type Artwork implements Node & Searchable & Sellable {
   # Based on artwork location verify that VAT info for the partner is complete.
   vatRequirementComplete: Boolean
 
+  # The visibility level of the artwork
+  visibilityLevel: String
+
```
Comment (jpotts244) [inline _schemaV2.graphql:2176]: Curious what others think here of typing?

I wonder if it is helpful at all to specify an enum type here (well in the artwork.index page) similar to how it is handled in [artwork import source](https://github.com/artsy/metaphysics/blob/main/src/schema/v2/artwork/index.ts#L323-L335) for example. Then it is a little more explicit in the schema that the values for visibilityLevel are expected to only be X,Y,Z. Example [here](https://github.com/artsy/metaphysics/pull/3886/files)

Comment (nspinazz89) [reply _schemaV2.graphql:2176]: So I based this on how we handle `cost_currency_code` which is also an enum field on `Artwork` in gravity, but definitely open to changing it!

File: src/schema/v2/artwork/index.ts (hunk 1 above)
Comment (nspinazz89) [reply src/schema/v2/artwork/index.ts:1453]: cool cool will do!

File: _schemaV2.graphql (hunk 2 above)
Comment (ansor4) [reply _schemaV2.graphql:2176]: blocking: +1 to adding an enum for this. Part of adding an enum to a graphql schema is that client will know that the field is an enum and not just a string. I can see why we might have done that for `cost_currency_code`, but honestly that should be a graphql enum as well.

**Review by ansor4 — CHANGES_REQUESTED** [review]
Let's create the enum that @jpotts244 mentioned! Happy to pair/mob on it!

File: _schemaV2.graphql (hunk 2 above)
Comment (nspinazz89) [reply _schemaV2.graphql:2176]: alrighty I'm convinced and I switched it to an ENUM!

**Review by ansor4 — APPROVED** [review]
thanks!

**Pull request merged**